source/modulo_components/include/modulo_components/ComponentInterface.hpp

```diff
@@ -48,26 +48,27 @@ class ComponentInterface : NodeT {
   void add_predicate(const std::string& predicate_name, const std::function<bool(void)>& predicate_function);
 
   /**
-   * @brief Get the value of the predicate given as parameter
+   * @brief Get the value of the predicate given as parameter, if the predicate is not found or the callable function
+   * fails return false
```

Comment:
Avoid run-on sentences - if the brief description is longer than a sentence, use `@details`. I also would avoid "the predicate given as a parameter" in favour of something like "a / the named predicate", or even just "Get the logical value of a predicate". It's not necessary to specify "the passed parameter" for single-argument methods.

This would apply to the other docstrings here and in Python as well.

```suggestion
   * @brief Get the logical value of a predicate.
   * @details If the predicate is not found or the callable function fails, the return value is false.
```